storage/src/vespa/storage/storageserver/statemanager.cpp

```diff
@@ -478,6 +485,28 @@ StateManager::run(framework::ThreadHandle& thread)
 
 }
 
+void
+StateManager::warn_on_missing_health_ping()
+{
+    vespalib::steady_time now(vespalib::steady_clock::now());
+    std::optional<vespalib::steady_time> health_ping_time;
+    {
+        std::lock_guard lock(_stateLock);
+        if (now <= _health_ping_warn_time) {
+            return;
+        }
+        health_ping_time = _health_ping_time;
+        _health_ping_warn_time = now + _health_ping_warn_interval;
+    }
+    if (health_ping_time.has_value()) {
+        vespalib::duration duration = now - health_ping_time.value();
+        LOG(warning, "Last health ping was %1.1f seconds ago", vespalib::to_s(duration));
+    } else {
+        vespalib::duration duration = now - _start_time;
+        LOG(warning, "No health pings since startup %1.1f seconds ago", vespalib::to_s(duration));
```

Comment:
Maybe we should explicitly state the service, i.e. `"No health pings from cluster controller since ..."`/`"Last health ping from cluster controller was ..."` in these log messages?

Reply:
```
n 0.027 seconds
[2023-07-19 16:23:52.512] INFO    container-clustercontroller Container.com.yahoo.container.jdisc.ConfiguredApplication	Destroy: Finished
[2023-07-19 16:27:19.890] WARNING distributor      vds.state.manager	Last health ping was 300.5 seconds ago
[2023-07-19 16:27:26.427] WARNING searchnode       proton.state.manager	Last health ping was 300.3 seconds ago
[2023-07-19 16:32:19.914] WARNING distributor      vds.state.manager	Last health ping was 600.5 seconds ago
[2023-07-19 16:32:26.450] WARNING searchnode       proton.state.manager	Last health ping was 600.3 seconds ago
```